### PR TITLE
Implement get_physical_placement command

### PR DESCRIPTION
**Pull request by DenverM80**
Also, update to use ds3-sdk:3.0.0-RC2

### CODE REVIEW

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetService.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public CliCommand init(final Arguments args) throws Exception {
     @Override
     public GetServiceResult call() throws IOException, SignatureException, SSLSetupException, CommandException {
         try {
-            final GetBucketsResponse response = getClient().getBuckets(new GetBucketsRequest()); // TODO revert to GetServiceRequest once the SDK is updated.
+            final GetServiceResponse response = getClient().getService(new GetServiceRequest()); // TODO revert to GetServiceRequest once the SDK is updated.
```
Comment (RachelTucker) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetService.java]: This comment can be removed now

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetPhysicalPlacement.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package com.spectralogic.ds3cli.command;
```
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetPhysicalPlacement.java]: missing copy rights

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetPhysicalPlacement.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    @Override
+    public CliCommand init(final Arguments args) throws Exception {
+        this.bucketName = args.getBucket();
+        if (this.bucketName == null) {
+            throw new MissingOptionException("The get physical placement command requires '-b' to be set.");
+        }
+
+        this.objectName = args.getObjectName();
+        if (this.objectName == null) {
+            throw new MissingOptionException("The get physical placement command requires '-o' to be set.");
```
Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetPhysicalPlacement.java]: So we are limiting them to specifying a single object?

Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetPhysicalPlacement.java]: I asked Jeff in the JIRA task about customer use case and he said that's fine

Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetPhysicalPlacement.java]: Perfect.  Was just making sure.

Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetPhysicalPlacement.java]: We may want to display a warning if they try to use -d

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/models/GetPhysicalPlacementWithFullDetailsResult.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.spectralogic.ds3cli.models;
```
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/models/GetPhysicalPlacementWithFullDetailsResult.java]: missing copy rights

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetPhysicalPlacementWithFullDetailsView.java [hunk 5]
```diff
@@ -0,0 +1,134 @@
+package com.spectralogic.ds3cli.views.cli;
```
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetPhysicalPlacementWithFullDetailsView.java]: copy rights

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetPhysicalPlacementWithFullDetailsView.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package com.spectralogic.ds3cli.views.json;
```
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetPhysicalPlacementWithFullDetailsView.java]: copy rights

**Comment by DenverM80** [comment]
updated

**Pull request merged**